internal/compiler/generator/cpp.rs

```diff
@@ -3553,6 +3553,12 @@ fn compile_builtin_function_call(
             ctx.generator_state.conditional_includes.cstdlib.set(true);
             format!("[](const auto &a){{ float res = 0; slint::cbindgen_private::slint_string_to_float(&a, &res); return res; }}({})", a.next().unwrap())
         }
+        BuiltinFunction::StringIsEmpty => {
+            format!("{}.empty()", a.next().unwrap())
+        }
+        BuiltinFunction::StringCharacterCount => {
+            format!("[](const auto &a){{ return slint::cbindgen_private::slint_string_character_count(&a); }}({})", a.next().unwrap())
```

Comment:
```suggestion
            format!("slint::cbindgen_private::slint_string_character_count({})", a.next().unwrap())
```

I think there is no need to introduce a scope here

Reply:
Thanks for your review. I have tried the code you proposed but it doesn't work because it generates `slint::cbindgen_private::slint_string_character_count(slint::SharedString(u8""))` while `slint_string_character_count()` takes `string: &SharedString`.

Reply:
Ah indeed. never mind then.